index.html

```diff
@@ -3246,6 +3246,55 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of <a>properties</a> serving as possible extension points. While
+some implementers signaled interest in these, their inclusion in this specification
+was considered to be premature; these extension points might be more
+formally defined in future versions of this specification. While these extension
+points are reserved in the base context, their usage is undefined by this
+specification and implementers are cautioned that usage of these properties is
+considered experimental.
+        </p>
+
+        <p class="issue" title="Reserved properties under debate">
+The following extension point properties are under consideration for being
+marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
+`evidence`, `render`, and `confirmationMethod`.
```

Comment:
Consider putting these items sorted in a `<ul>`.

Reply:
> Consider putting these items sorted in a `<ul>`.

These are in an issue marker. This PR is for the concept of having reserved extension points. If this PR goes through, then we'll create a table of extension points with possibly some descriptions of the expected purpose of the extension point along with links to the VC-SPECS-DIR section on each extension point.